### ISSUE TITLE
ZE.8001 fire even if you are in the same faction

### ISSUE THREAD

**Issue by MemeWizard** [Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth#1137]
**Your mod version is:**
Master Branch

**What expansions do you have installed?**
All

**Are you using any submods/mods? If so, which?**
Guardians of Azeroth

**Please explain your issue in as much detail as possible:**
ZE.8001 can fire even if you are in the same faction

**Steps to reproduce the issue:**
???

**Upload an attachment below: .zip of your save, or screenshots:**

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/73866356-c0a9e080-4855-11ea-87d5-2c9dc2f517cb.png)

</details>

**Comment by zumbak04** [comment]
Don't get it. Non-aggression pact of faction members lasts 1 year. After that, you can re-sign it.

**Comment by MemeWizard** [comment]
Don't get it.
You can't attack alliance members anyway and non-aggression pact is broken when one of the side leave the faction.

**Comment by zumbak04** [comment]
If you think it's weird signing non-aggression pact with an ally, I'll mark it as Paradox bug.

**Comment by MemeWizard** [comment]
> I'll mark it as Paradox bug.

Non-aggression pact with an ally in vanilla make sense, because you can attack your allies

**Comment by zumbak04** [comment]
Aight, make sense.
